### PR TITLE
NEWS: News update before RC3

### PR DESCRIPTION
**Pull request by shamisp**
RC3 release

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -11,6 +11,9 @@
 ### Features:
 ### Bugfixes:
 
+## 1.14.1 RC3 (May 19, 2023)
+* Fixed ROCM to prevent the locking of host pinned memory
```
Comment (yosefe) [inline NEWS:15]: ROCm

Comment (yosefe) [reply NEWS:15]: otherwise, LGTM

**Review by pentschev — APPROVED** [review]
LGTM after Yossi's comment is fixed.

**Comment by shamisp** [comment]
The patch was approved  offline.

**Pull request merged**